flatpak/org.nanuc.Axolotl.appdata.xml

```diff
@@ -21,6 +21,9 @@
         </p>
     </description>
     <releases>
+      <release version="0.8.9" date="2020-11-30">
```

Comment:
It should be 0.9.0 there.

Reply:
i missed the appimage, too. It's the first release also for flatpack and appimage so it's new where to change the number :)